Regression in pyplot / samples

<!-- Please fill in the following information to report a problem with Cantera. If you have a question about using Cantera, please post it on our Google Users' Group (https://groups.google.com/forum/#!forum/cantera-users). Feature enhancements should be discussed in the dedicated Cantera enhancements repository (https://github.com/Cantera/enhancements/new/choose) -->

**Problem description**

<!-- A clear and concise description of what the bug is. -->

An upstream update presumably broke the samples GH test runners (see #1531 or #1262)

**Steps to reproduce**

<!-- A minimal, complete, and reproducible example demonstrating the problem. See https://stackoverflow.com/help/minimal-reproducible-example for additional suggestions on how to create such an example. -->

1. Use GH Action to test samples
2. See error '....'

```
[...]
Traceback (most recent call last):
  File "samples/python/kinetics/mechanism_reduction.py", line 95, in <module>
    plt.tight_layout()
  File "/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/matplotlib/pyplot.py", line 2349, in tight_layout
    return gcf().tight_layout(pad=pad, h_pad=h_pad, w_pad=w_pad, rect=rect)
  File "/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/matplotlib/figure.py", line 3547, in tight_layout
    _api.warn_external('The figure layout has changed to tight')
  File "/opt/hostedtoolcache/Python/3.8.17/x64/lib/python3.8/site-packages/matplotlib/_api/__init__.py", line 390, in warn_external
    warnings.warn(message, category, stacklevel)
UserWarning: The figure layout has changed to tight
Error: Process completed with exit code 1.
```

**Behavior**

<!-- Describe the result of executing the above steps, and how this differs from what you expect to happen. -->

The `UserWarning` crashes the GH Action.

**System information**

- Cantera version: 3.0.0b1
- OS: GH Runner / Ubuntu 20.04 and 22.04

--- a/samples/python/kinetics/mechanism_reduction.py
+++ b/samples/python/kinetics/mechanism_reduction.py
@@ -92,6 +92,6 @@ for i, N in enumerate([40, 50, 60, 70, 80]):
     plt.title('Reduced mechanism ignition delay times\n'
               'K: number of species; R: number of reactions')
     plt.xlim(0, 20)
-    plt.tight_layout()
 
+plt.tight_layout()
 plt.show()

--- a/samples/python/reactors/sensitivity1.py
+++ b/samples/python/reactors/sensitivity1.py
@@ -63,9 +63,9 @@ if '--plot' in sys.argv:
     plt.tight_layout()
 
     plt.figure(2)
-    plt.plot(states.t, states.s2, '-', states.t, states.s3, '-g')
-    plt.legend([sim.sensitivity_parameter_name(2), sim.sensitivity_parameter_name(3)],
-               'best')
+    plt.plot(states.t, states.s2, '-', label=sim.sensitivity_parameter_name(2))
+    plt.plot(states.t, states.s3, '-g', label=sim.sensitivity_parameter_name(3))
+    plt.legend(loc='best')
     plt.xlabel('Time (ms)')
     plt.ylabel('OH Sensitivity')
     plt.tight_layout()

